src/sidebar.js

```diff
@@ -66,7 +66,6 @@ module.exports = {
           label: 'Passwordless',
           items: [
             'guides/passwordless/webauthn'
-            // 'guides/passwordless/social-sign-in'
```

Comment:
So... This was there and I think YOU added it :D 
https://github.com/ory/docs/pull/654/files#diff-c939543b5c9b8c7d51be34c77a7a47d2cdf59f173cf99fe9fad036d5b8c9ed71

As for the redirect - sure, I can add it, but is it necessary for a document that was never listed in the sidebar and was empty at the same time?

Reply:
> As for the redirect - sure, I can add it, but is it necessary for a document that was never listed in the sidebar and was empty at the same time?

Depends, if it was indexed in google it should receive one, otherwise we'll get a penalty! Looking at Google, it was indeed indexed:

<img width="726" alt="Bildschirmfoto 2022-05-10 um 13 38 24" src="https://user-images.githubusercontent.com/3372410/167620053-24f23627-4c81-4b31-bee7-bd3937463adb.png">

Reply:
> So... This was there and I think YOU added it :D

😁

Reply:
Mmmmm, looks spectacular in google :D Adding a redirect then!